nc_time_axis/__init__.py

```diff
@@ -233,14 +248,20 @@ def convert(cls, value, unit, axis):
                 return value
             first_value = value
 
-        if not hasattr(first_value, 'calendar'):
-            raise ValueError('A "calendar" attribute must be attached to '
-                             'netcdftime object to understand them properly.')
-        return netcdftime.date2num(value, cls.standard_unit,
-                                   calendar=first_value.calendar)
+        if not isinstance(first_value.datetime, netcdftime.datetime):
```

Comment:
I think I was counting (incorrectly) on the fact that someone would never use `NetCDFTimeConvert` directly, that it was only there for matplotlib to make use of. Since `NetCDFTimeConvert` had been registered with in the `munits.regristry` with the object `CalendarDateTime`, I thought I could assume that a `CalendarDateTime` object would only ever be passed to this.
I think I'm wrong in that assumption so I'm going to add a check that it is of type `CalendarDateTime`.